build.fsx

```diff
@@ -393,8 +393,10 @@ let callPaket wd args =
 /// <param name="dllPath">Test assembly to run tests from</param>
 /// <param name="resultsXml">Expecto test results XML file</param>
 let runExpecto workDir dllPath resultsXml =
+    let resultsFile = "testresults" </> resultsXml
```

Comment:
As it stands, the test result xml files get written into the root directory of the repository, but it's easier to upload them in a batch if they all get written into a subdirectory and this looked like the easiest place to change to do that